website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,412 @@
+  }
+ // This will trigger 'notification' and 'localNotification' events on PushNotificationIOS
+  [RCTPushNotificationManager didReceiveNotification:response.notification];
+  completionHandler();
+}
+```
+
+Finally, delete the following methods and adapt the logic into the callbacks above which will be called instead:
+
+1. <code>[application:didReceiveLocalNotification:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622930-application?language=objc)</code> [deprecated]
+2. <code>[application:didReceiveRemoteNotification:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623117-application?language=objc)</code> [deprecated]
+3. <code>[application:didReceiveRemoteNotification:fetchCompletionHandler:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623013-application?language=objc)</code> [not deprecated, but is superseded by the <code>UNUserNotificationCenterDelegate</code> methods]
+
+Delete any usages of <code>[application:didRegisterUserNotificationSettings:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623022-application?language=objc)</code> and <code>RCTPushNotificationManager</code>’s corresponding <code>didRegisterUserNotificationSettings:</code> as well.
+
+**Example:** See the RNTester <code>[AppDelegate.mm](https://github.com/facebook/react-native/blob/main/packages/rn-tester/RNTester/AppDelegate.mm)</code>.
+
+#### JS
+
+1. Remove any references to `alertAction`.
+2. Remove the `handler` argument on any calls to `removeEventListener`.
+3. Replace any usages of `repeatInterval` by firing multiple notifications using `fireDate` or `fireIntervalSeconds` instead.
+4. Note that `soundName` will be null when it is accessed on a `Notification` returned from `getScheduledLocalNotifications()` and `getDeliveredNotifications()`.
+
+</details>
+
+### Android SDK minimum bump (Android 6.0)
+
+React Native 0.74 has a minimum SDK version requirement of 23 (Android 6.0). See our justification for this change [here](https://github.com/react-native-community/discussions-and-proposals/discussions/740).
+
+### Yarn 3 for New Projects
+
+[Yarn 3](https://yarnpkg.com/blog/release/3.0) is now the default JavaScript package manager for new projects initialized with React Native Community CLI.
+
+Yarn 3.x will be used with `nodeLinker: node-modules`, a mode providing compatibility with React Native libraries. This replaces Yarn Classic (1.x, deprecated) as the previous default. To upgrade Yarn version inside your existing app you can follow this [guide](https://yarnpkg.com/migration/guide).
+
+```sh
+$ yarn --help
+━━━ Yarn Package Manager - 3.6.4 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  $ yarn <command>
+```
+
+The Community CLI also supports initializing projects with other package managers via the `--pm` flag ([read more](https://github.com/react-native-community/cli/blob/main/docs/init.md)).
+
+## Breaking Changes
```

Comment:
@thymikee @szymonrybczak Do we have any breaking RN CLI changes to highlight?

Reply:
Nothing other than Yarn 3 that's already mentioned